pkg/api/dashboard.go

```diff
@@ -33,6 +33,12 @@ func (g DashboardHandler) Append(router *mux.Router) {
 		Handler(http.StripPrefix("/dashboard/", http.FileServer(g.Assets)))
 }
 
+func (g DashboardHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	// allow iframes from our domains only
+	w.Header().Set("Content-Security-Policy", "frame-ancestors 'self' traefik.io *.traefik.io;")
```

Comment:
Is this something that we could configure with a default? I'm just thinking worst case scenario, if something happened to those records, users could temporarily override with a CLI arg, rather than having to use a rebuilt binary/image.